day1/1/my_try_cnt.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+from collections import Counter 
+
+def word_list(f):
+    
+    cnt = Counter()
```

Comment:
Guardare il costruttore di `Counter`

Reply:
Forse c'è un modo per inizializzare cnt in una sola riga con le parole del file, ma non ci sono riuscito

ho provato cose del tipo: cnt=counter(f.readilines.split()) ma ovviamente non c'è il metodo split per readlines

Reply:
```
readlines => ["ciao bao", "pipplo pluto"]
[["ciao", "bao"], ["pippo", "pluto"]]
["ciao", "bao", "pippo", "pluto"]
```

Trovare il modo per rendere una lista di liste una lista flat

Reply:
hint: guardare `itertools`